core/account.py

```diff
@@ -0,0 +1,76 @@
+"""
+Contains the functions directly used by the openapi spec.
+"""
+from typing import Tuple
+
+import shortuuid
+from core.data_operations.account_data import check_exists_then_get
+from core.data_operations.account_data import get_data_by_email
+from core.db.db_connection import db_connection
+
+
+def get_account(
+    email_address: str = None, account_id: str = None
+) -> Tuple[dict, int]:
+    """get_account Given an email or account id, the corresponding
+    entry in the db is returned.
+
+    Args:
+        email_address (str, optional): An email address given as a string.
+        Defaults to None.
+        account_id (_type_, optional): An account_id given as a string.
+        Defaults to None.
+
+    Returns:
+        dict, int
+    """
+
+    # print(f"email : {email_address} + account_id {account_id}")
+
+    if email_address is None and account_id is None:
```

Comment:
is this a new line between everything an agreed style for us? when the code doesnt logically clump together I find it harder to read.

Reply:
generally i'd clump variables, logic and return related. For smaller methods I'd clump everything. Logic can be separated into multiple logical clumps when it makes logical sense to do so

Reply:
Typically I do clump logical parts together - my fault for turning off line numbers in my IDE. Now fixed :)